embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/envelope/resource/Device.kt

```diff
@@ -0,0 +1,286 @@
+package io.embrace.android.embracesdk.capture.envelope.resource
+
+import android.os.Build
+import android.os.Environment
+import android.os.StatFs
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import androidx.annotation.ChecksSdkIntAtLeast
+import io.embrace.android.embracesdk.capture.cpu.CpuInfoDelegate
+import io.embrace.android.embracesdk.logging.InternalStaticEmbraceLogger.Companion.logDebug
+import io.embrace.android.embracesdk.logging.InternalStaticEmbraceLogger.Companion.logDeveloper
+import io.embrace.android.embracesdk.prefs.PreferencesService
+import io.embrace.android.embracesdk.worker.BackgroundWorker
+import java.io.File
+import java.util.Locale
+
+public interface Device {
```

Comment:
This and all its fields can be `internal` unless there's a good reason to expose any of this